backend/create_database.py

```diff
@@ -21,9 +21,10 @@ def db_create():
         create_database(db_credentials)
         db.create_all()
         db.session.commit()
-        print "Successfully created database and tables."
+        print("Successfully created database and tables.")
```

Comment:
`print` in Python 2.x is not function, but statement. Please remove parentheses.

Reply:
+